src/app/containers/Header/index.jsx

```diff
@@ -22,7 +23,9 @@ const HeaderContainer = () => {
   // All other page types show the nav bar at all times
   const showNav = showNavOnArticles || pageType !== 'article';
 
-  const skipLink = (
+  const isOperaMini = onClient() && window.operamini;
```

Comment:
will this only effect operamini extreme mode or just opera mini in general?

Reply:
Good old StackOverflow has the answers - https://stackoverflow.com/questions/39387820/detecting-versions-of-opera-mini-extreme-mode-with-javascript. According to the comment and a linked Opera Mini blog, `window.operamini` will only be true when it's using the `extreme` mode. In Extreme mode I'm unable to use Android Talkback to go through all the elements on the page anyway, so removing the skip link altogether seemed like the best idea. This will need to be tested with an iOS device and possibly other versions of Android.

Reply:
Awesome, thanks for confirming. A nice future enhancement would be to add this value to one of our context providers, that way any component would be able to use it 👍 One a separate PR

Reply:
Nice finding @jblaut. I've been googling it and in this post https://dev.opera.com/articles/opera-mini-and-javascript/ (very old), they are suggesting to use this code:
`const isOperaMini = Object.prototype.toString.call(window.operamini) === "[object OperaMini]`.
Not sure if it is safer to do it in that way though.

Reply:
@DenisHdz it looks like if we were to use that piece of code the results for Opera Mini and non-Opera would be `[object OperaMini]` and `[object Undefined]`, which means we would be comparing strings. 

![image](https://user-images.githubusercontent.com/13763899/78007440-34d4a200-7336-11ea-8298-1db755ebbfe4.png)

Currently we're comparing Boolean values, which is probably safer